count-stats/src/main/java/uk/ac/ebi/eva/countstats/model/Count.java

```diff
@@ -0,0 +1,37 @@
+package uk.ac.ebi.eva.countstats.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.hibernate.annotations.TypeDefs;
+import uk.ac.ebi.eva.countstats.configuration.StringJsonUserType;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "count_stats")
+@TypeDefs({@TypeDef(name = "StringJsonObject", typeClass = StringJsonUserType.class)})
+public class Count {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NotNull
+    private String process;
+    @NotNull
+    @Type(type = "StringJsonObject")
+    private String identifier;
+    @NotNull
+    private String metric;
+    @NotNull
+    private long count;
```

Comment:
Sorry for the late request but I was wondering if we can also add a timestamp for easy tracking...

Reply:
added